### ISSUE TITLE
Add Basic Engine Analysis

### ISSUE THREAD

**Issue by Kozmocha** [UNCG-CSE/Volvo_Truck_Analytics#19]
Look at:
- EngineSpeed
- BoostPressure
- RoadInclinationVP15
- ActualEnginePercentTorque
- ActualEnginePercentTorqueHiRes 

Place them all on one graph per day. All are going to need their own Axis because the value ranges are different.

**Edit: this task is dependent on #26.**

**Comment by Kozmocha** [comment]
Placing this issue on hold due to current time-constraints.

**Comment by Kozmocha** [comment]
Projected timeline does not leave enough time to complete a basic engine analysis. Closing Issue for now.
